schemas/variant_processing.json

```diff
@@ -0,0 +1,126 @@
+{
+  "name": "variant_processing",
+  "schema": {
+    "type": "object",
+    "required": [
+      "workflow",
+      "experiment",
+      "variant_class"
+    ],
+    "properties": {
+      "workflow": {
+        "type": "object",
+        "required": [
+          "workflow_name",
+          "workflow_short_name",
+          "workflow_version",
+          "genome_build",
+          "run_id",
+          "session_id",
+          "inputs"
+        ],
+        "properties": {
+          "inputs": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "analysis_type": {
+                  "type": "string",
+                  "const": "variant_calling"
+                },
+                "input_analysis_id": {
+                  "type": "string",
+                  "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+                }
+              }
+            },
+            "minItems": 1
```

Comment:
the schema is much the same as `variant_calling` expect for the inputs of the workflow part. Was thinking whether we should add `maxItems: 1` here as well, maybe not, might be in the future we could have variant processing analysis taking more than one input.

Reply:
that's right. like not sure if `consensus variant calls` should use `variant_calling` or `variant_processing`?